expr/literals.go

```diff
@@ -84,6 +89,8 @@ type Literal interface {
 	ToProto() *proto.Expression
 	ToProtoLiteral() *proto.Expression_Literal
 	Visit(VisitFunc) Expression
+	// ValueString returns a string representation of the literal's value only.
```

Comment:
we should probably expand this description a little bit in how we explain that it'll return timestamps/dates/etc. instead of the raw underlying value

Reply:
done.  can tweak more if desired